express/src/controllers/manage-account.ts

```diff
@@ -56,7 +56,6 @@ export const processAddServiceForm = async function (req: Request, res: Response
     let newServiceData = JSON.parse(newServiceOutput?.data.output);
     const generatedClient = await lambdaFacade.generateClient(newServiceData.serviceId, service, newUser.email as string, req.session.authenticationResult?.AccessToken as string)
     const body = JSON.parse(generatedClient.data.output).body;
-    const clientId = JSON.parse(body).clientId;
-
-    res.redirect(`/client-details/${clientId}`);
+    const serviceId = JSON.parse(body).pk;
+    res.redirect(`/client-details/${serviceId.substring(8)}`);
```

Comment:
Why `.substring(8)`?

Reply:
When we get the service ID back from the DB it's prepended with `service#`.  The `#` breaks path parameters if we stick it straight in so we have to take it off.  When we tidy all of this up and refactor this should definitely made 'clean' and self-documenting.  I'm kind of racked with guilt for not at least using `const LENGTH_OF_SERVICE_PREFIX = 8;` or something.

Reply:
Or just split on `#`